modules/special-rules/pages/armour-1.adoc

```diff
@@ -0,0 +1,9 @@
+= Armour +1
+
+These units have particularly heavy armour -- such as Ork Meganobz.
+
+---
+
+Apply this rule to units that have the {armour-1} ability, relative to their base units:
```

Comment:
I'd leave this for now, but as an aside while other related parts of our conversation are fresh in mind:

This type of situation is where a single lonely bullet in some places is the lesser evil, because otherwise it gets messy with the "relative to their base units" qualifier.
That said, this 'singular unit' construction is perhaps better:
```suggestion
Where a unit has the {armour-1} ability, apply this rule relative to their base unit:
```
But I think we can look back over all the constructions in another future pass to unify them. I agree that we should try to be somewhat 'consistent in our inconsistency', but we shouldn't try to predict and enforce a single construction too rigidly at this stage.

Reply:
Thinking about it further, 'apply this **change** relative to their base unit' would be even better, and would help to further differentiate between profile modifications and in-game rules.

Reply:
The last suggestion ('apply this change') added to commit batch.